build/x86_64_debian/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apt-get -y update \
         make \
         ruby \
  # RVM GPG keys.
- && gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+ && gpg2 --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
```

Comment:
Do we not need [the IPv6 config](https://github.com/GoogleCloudPlatform/google-fluentd/pull/249#discussion_r396134039) as well?

Reply:
Added.